Tests/Core/URLBuilderTests.swift

```diff
@@ -163,6 +163,8 @@ class URLBuilderTests: XCTestCase {
     )
   }
 
+  // MARK: - GraphRequest
```

Comment:
what's `MARK` mean? TODO ?

Reply:
It's part of the jump bar in Xcode. Allows for easier code navigation. :)

![image](https://user-images.githubusercontent.com/4922820/61910430-84321b00-aee9-11e9-98ad-8fb5b4c00d84.png)